appmap/test/data/agent_status/appmap.yml

```diff
@@ -0,0 +1,9 @@
+name: agent_status_test
+packages: []
+test_commands:
```

Comment:
Is this just a small hack for Python or we going to update the docs and other language agents as well? /cc @kgilpin